src/ovirtapi.js

```diff
@@ -267,8 +269,13 @@ OvirtApi = {
         id: vm.cluster.id,
       } : undefined,
 
-      os: vm.os && vm.os.type ? {
-        type: vm.os.type,
+      os: vm.os && (vm.os.type || vm.os.bootDevices) ? {
+        type: vm.os.type || undefined,
+        boot: {
+          devices: {
+            device: vm.os.bootDevices.filter((item) => item !== null),
```

Comment:
weird case, but might happen: `vm.os.bootDevices === undefined`

Reply:
Actually no, because it get data from VmDialog, and VmDialog in any case will return array.